src/test/compile-fail/issue-60989.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
```

Comment:
Also -- don't we need an `.stderr` file?

Reply:
only ui tests need those

Reply:
oh